Subject: Simplify base64 dependency.

Debian currently has rust-base64 version 0.21.7.  Updating debian to
rust-base64 0.22 potentially affects dozens of packages.

The debian build daemons don't really handle "or"ed build-deps safely:
they choose the first one, and ignore the rest, which means that
they'll take the highest-value.  So, to simplify the update in debian,
we tighten this dependency.  if rust-base64 0.22 lands in debian, this
patch should be removed.

See https://bugs.debian.org/1069207

Index: sequoia-openpgp/Cargo.toml
===================================================================
--- sequoia-openpgp.orig/Cargo.toml
+++ sequoia-openpgp/Cargo.toml
@@ -72,7 +72,7 @@ harness = false
 version = "1.0.18"
 
 [dependencies.base64]
-version = ">= 0.21, < 0.23"
+version = "0.21"
 
 #[dependencies.block-padding]
 #version = "0.3"
